app/src/main/java/org/wikipedia/readinglist/db/ReadingListPageDao.kt

```diff
@@ -130,7 +136,7 @@ interface ReadingListPageDao {
         updateThumbAndDescriptionByName(pageProto.lang, pageProto.apiTitle, thumbUrl, description)
     }
 
-    fun findPageInAnyList(title: PageTitle): ReadingListPage? {
+    suspend fun findPageInAnyList(title: PageTitle): ReadingListPage? {
         return getPageByParams(
```

Comment:
The `getPageByParams` function also needs to be `suspend`, since that's where Room actually does its job.